Source/WebCore/rendering/style/StyleRareInheritedData.h

```diff
@@ -230,7 +230,8 @@ class StyleRareInheritedData : public RefCounted<StyleRareInheritedData> {
 #endif
     Style::ListStyleType listStyleType;
     Style::BlockEllipsis blockEllipsis;
-
+    PREFERRED_TYPE(bool) unsigned enableEvaluationTimeZoom : 1;
+    PREFERRED_TYPE(bool) unsigned useSVGZoomRulesForLength : 1;
```

Comment:
I can see why @anttik would suggest `enableEvaluationTimeZoom` be inherited (it is going to be the same for the every RenderStyle in a document) but I don't think you want `useSVGZoomRulesForLength` to be inherited, given that every RenderStyle needs this calculated independently based on the element.